core-bundle/contao/drivers/DC_Table.php

```diff
@@ -4753,7 +4754,7 @@ protected function parentView()
 						$security = System::getContainer()->get('security.helper');
 
 						// Create new button
-						if (!($GLOBALS['TL_DCA'][$this->strTable]['config']['closed'] ?? null) && !($GLOBALS['TL_DCA'][$this->strTable]['config']['notCreatable'] ?? null) && $security->isGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new CreateAction($this->strTable, array('pid' => $row[$i]['id']))))
+						if (!($GLOBALS['TL_DCA'][$this->strTable]['config']['closed'] ?? null) && !($GLOBALS['TL_DCA'][$this->strTable]['config']['notCreatable'] ?? null) && $security->isGranted(ContaoCorePermissions::DC_PREFIX . $this->strTable, new CreateAction($this->strTable, array('pid' => $row[$i]['pid'], 'sorting' => $row[$i]['sorting'] + 1))))
```

Comment:
What did you add the `sorting` value for? It is not handled in the `CreateAction` as far as I see.

Reply:
It is meant to tell where the new record would be added, but we currently don't need that. You could probably check if something can be added after something, if anyone has a case to deny that.

Reply:
I‘m not a big fan of adding things that are not used. Can we omit this until we have a use case?

Reply:
we have the same thing in other places. And just because the core does not need it, does not mean its not useful for anyone implementing permissions.

Reply:
As discussed in the Contao call, we should also provide the `ptable` value.